tools/downloader/topologies_test.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import os
+import numpy as np
+import cv2 as cv
+
+import topologies as omz
+
+from pathlib import Path
+from parameterized import parameterized
+from openvino import inference_engine
+from openvino.inference_engine import IECore
+
+class TestTopologies(unittest.TestCase):
+    @parameterized.expand(['FP16', 'FP32'])
```

Comment:
I don't think testing both of these does any good. The code path is exactly the same; you're not gaining any more coverage, but you are increasing the run time.